assets/shell-quote.js

```diff
@@ -25,6 +25,7 @@ var META = '|&;()<> \\t';
 var BAREWORD = '(\\\\[\'"' + META + ']|[^\\s\'"' + META + '])+';
```

Comment:
@SahilChoudhary22 This is an external asset that we have added to the module. We will have to be completely sure before we make any changes to it.